doc/tutorials/merge.md

```diff
@@ -32,29 +32,30 @@ It's worth noting, resultpath should be empty before attempting a merge, otherwi
 
 As for the options, there are a few basic options:
 
-	-i  --interactive		which attempts the merge in an interactive way
+	-i  --interactive	which attempts the merge in an interactive way
 
-	-t  --test			which tests the proposed merge and informs you about possible conflicts
+	-t  --test		which tests the proposed merge and informs you about possible
+				conflicts
 
-	-f --force			which overwrites any Keys in resultpath
+	-f --force		which overwrites any Keys in resultpath
 
 ### Strategies ###
 
 Additionally there is an option to specify a merge strategy, which is very important.
 
 The option for strategy is:
 
-	-s --strategy <name>		which is used to specify a strategy to use in case of a conflict
```

Comment:
I am pretty sure I looked at the syntax of the text. However, since I was mostly researching for issue #1019, I did not really check the semantics of the text.